src/verse/BaseVerseFormatter.ts

```diff
@@ -59,7 +59,7 @@ export abstract class BaseVerseFormatter {
         text +=
           ' ' + verseNumberFormatted + verse.text.trim().replaceAll('\n', ' ')
       } else {
-        text += '> ' + verseNumberFormatted + verse.text.trim() + '\n'
+        text += '> ' + verseNumberFormatted + verse.text.replace(/\r\n|\n|\r/g, " ") + "\n" // Remove extraneous line breaks in KJV verses.
```

Comment:
Shouldn't it be like this?

```suggestion
        text += '> ' + verseNumberFormatted + verse.text.trim().replace(/\r\n|\n|\r/g, " ") + "\n" // Remove extraneous line breaks in KJV verses.
```

Reply:
After looking into the `.trim()` method a little bit more, I can see why you might want to use both, as `.trim()` handles spaces and tabs as well as line breaks, which we can't do with `.replace()`. After some brief testing, it doesn't look like it changes much, but it doesn't hurt to add it.